EDA_analysis.Rmd

```diff
@@ -0,0 +1,80 @@
+---
+title: "Exoloratory Data Analysis of the Covid-19 Data Set"
+output: github_document
```

Comment:
Can you add` bibliography: doc/covid19.bib` under `output: github_document`, plese?